medusa/server/api/v2/scene_exception.py

```diff
@@ -0,0 +1,160 @@
+# coding=utf-8
+"""Request handler for scene_exceptions."""
+
+from medusa.scene_exceptions import get_last_refresh, retrieve_exceptions
+from tornado.escape import json_decode
+from .base import BaseRequestHandler
+from .... import db
+
+
+def get_last_updates():
+    """Query the cache table for the last update for every scene exception source."""
+    last_updates = {}
+    for scene_exception_source in ['custom_exceptions', 'xem', 'anidb']:
+        last_updates[scene_exception_source] = get_last_refresh(scene_exception_source)[0]['last_refreshed']
+    return last_updates
+
+
+class SceneExceptionOperationHandler(BaseRequestHandler):
+    """Scene Exception request handler."""
+
+    def set_default_headers(self):
+        """Set default CORS headers."""
+        super(SceneExceptionOperationHandler, self).set_default_headers()
+        self.set_header('Access-Control-Allow-Methods', 'PUT')
+
+    def put(self):
+        """Start fetch retrieving scene name exceptions."""
+        json_body = json_decode(self.request.body)
+
+        if json_body.get('type', '') == 'REFRESH':
+            retrieve_exceptions(force=True)
+            return self.api_finish(status=201)
+        return self.api_finish(status=400)
+
+
+class SceneExceptionHandler(BaseRequestHandler):
+    """Shows request handler."""
+
+    def set_default_headers(self):
+        """Set default CORS headers."""
+        super(SceneExceptionHandler, self).set_default_headers()
+        self.set_header('Access-Control-Allow-Methods', 'GET, OPTIONS, PUT, POST')
+
+    def get(self, *args, **kwargs):
+        """Query scene_exception information.
+
+        :param show_indexer_id: The indexer id. Like 1 for tmdb and 3 for tvmaze.
+        :param show_id:
+        :type show_indexer_id: str
+        :param season:
+        """
+        exception_id = self._parse(kwargs.pop('row_id'))
+        indexer = self.get_query_argument('indexer', None)
+        indexer_id = self.get_query_argument('indexer_id', None)
+        season = self.get_query_argument('season', None)
+        detailed = self.get_query_argument('detailed', 'true').lower()
+
+        if detailed == 'false':
+            return self.api_finish(data={"last_updates": get_last_updates()})
```

Comment:
GET sceneexception should always return sceneexception. Detailed will return with all info. Non detailed should return with just `header` info